android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListSectionProxy.java

```diff
@@ -234,6 +234,15 @@ public String getFooterTitle()
 		return footerTitle;
 	}
 
+	// clang-format off
+	@Kroll.method
+	@Kroll.getProperty
+	public int itemCount()
```

Comment:
Do we need this one as well? It looks incorrect to have no `get` as a prefix. Also the `getItemCount()` method already exists and could be extended by the `@Kroll.getProperty` annotation.

Reply:
let me check

Reply:
works fine! Remove the custom getter